server/index.js

```diff
@@ -1,42 +0,0 @@
-const express = require ('express')
-const bodyParser = require ('body-parser')
-const cors = require('cors')
-
-const app = express()
-
-app.use(bodyParser.json())
-app.use(cors())
-
-let animalArr = [
-    {
-      name: 'Armadillo',
-      id: '1',
-      imageUrl: 'https://upload.wikimedia.org/wikipedia/commons/thumb/b/b4/Nine-banded_Armadillo.jpg/1200px-Nine-banded_Armadillo.jpg'
-    },
-    {
-      name: 'Tiger',
-      id: '2',
-      imageUrl: 'http://kids.sandiegozoo.org/sites/default/files/2017-06/animal-hero-tiger_0.jpg'
-    }
-  ]
-
-  app.get('/api/animals', (req, res) => {
-      res.send(animalArr)
-      console.log('get request')   
-  })
-
-  app.post('/api/animals', (req, res) => {
-      animalArr.push(req.body)
-      console.log('animalArr', animalArr)
-      res.send(animalArr)
```

Comment:
Here I wanted the server to handle adding the `id` to each object, so something like this: 
```
const animalToAdd = req.body
animalToAdd.id = animalArr.length + 1 //this programmatically creates the id 
animalArr.push(animalToAdd)
```